scripts/ios/download_libs

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Again, can you add the extension? I think this one actually existed already with the .sh extension so it's probably just a matter of editing that one instead of adding a new scrtipt